sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## 1.9.0-beta.1 (Unreleased)
 
 ### Features Added
-
+* Upgrade `MessagePack` from 1.7.3.7 to 1.9.11 to fix a vulnerability.
```

Comment:
I wouldn't advertise it so plainly. Just "Upgraded MessagePack for performance and security improvements" is the usual fare.